mix.exs

```diff
@@ -67,10 +68,20 @@ defmodule Nimble.MixProject do
   defp aliases do
     [
       codebase: ["format --check-formatted", "credo", "sobelow --config"],
+      coverage: ["coveralls.html --raise"],
       "ecto.setup": ["ecto.create", "ecto.migrate", "run priv/repo/seeds.exs"],
       "ecto.reset": ["ecto.drop", "ecto.setup"],
-      test: ["ecto.create --quiet", "ecto.migrate", "test"],
-      coverage: ["coveralls.html --raise"]
+      test: [
+        "ecto.create --quiet",
+        "ecto.migrate",
+        "assets.compile --quiet",
+        "test"
+      ],
+      "assets.compile": &compile_assets/1
     ]
   end
+
+  defp compile_assets(_) do
+    Mix.shell().cmd("cd assets && ./node_modules/.bin/webpack --mode development", quiet: true)
```

Comment:
Let's add `build:dev` to `assets/package.json` instead?

Reply:
Added in https://github.com/nimblehq/elixir-templates/pull/17/commits/8f1421b68a8e0ab15635e4742d56f9490db2217a @Tolsee